xml/depl_nodes.xml

```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
+      <literal>NAMESPACE</literal> keyword is automatically replaced with the
+      appropriate network namespace for the VM being monitored. Set to
+      <parameter>False</parameter> to disable ping checks.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vnic_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting disk metrics.
+      Increase this value to reduce I/O load. If the value is lower than the
+      global agent collection period (<option>check_frequency</option>), it
+      will be ignored in favor of the global collection period.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_cpu_check_enable
+    </term>
+    <listitem>
+     <para>
+      Toggles the collection of VM CPU metrics. Set to
+      <parameter>true</parameter> to enable.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_disks_check_enable
+    </term>
+    <listitem>
+     <para>
+      Toggles the collection of VM disk metrics. Set to
+      <parameter>true</parameter> to enable.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_extended_disks_check_enable
+    </term>
+    <listitem>
+     <para>
+      Toggles the collection of extended disk metrics. Set to
+      <parameter>true</parameter> to enable.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_network_check_enable
+    </term>
+    <listitem>
+     <para>
+      Toggles the collection of VM network metrics. Set to
+      <parameter>true</parameter> to enable.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_ping_check_enable
+    </term>
+    <listitem>
+     <para>
+      Toggles ping checks for checking whether a host is alive. Set to
+      <parameter>true</parameter> to enable.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vm_probation
+    </term>
+    <listitem>
+     <para>
+      Specifies a period of time (in seconds) in which to suspend metrics from
+      a newly-created VM. This is to prevent quickly-obsolete metrics in an
+      environment with a high amount of instance churn (VMs created and
+      destroyed in rapid succession). The default probation length is 300
+      seconds (5 minutes). Set to 0 to disable VM probation. In this case,
+      metrics are recorded immediately after a VM is created.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     vnic_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting VM network
+      metrics. Increase this value to reduce I/O load. If the value is lower
+      than the global agent collection period
+      (<parameter>check_frequency</parameter>), it will be ignored in favor of
```

Comment:
```suggestion
(<parameter>check_frequency</parameter>), it is ignored in favor of
```

Reply:
1. For this and several other instances of `will`...These are conditional sentences, and `will` is appropriate.